roles/ceph-mon/tasks/start_monitor.yml

```diff
@@ -46,6 +46,14 @@
     ansible_distribution != "Ubuntu" and
     not is_ceph_infernalis
 
+- name: enable systemd unit file for mon instance (for or after infernalis)
+  command: "ln -s /usr/lib/systemd/system/ceph-mon@.service /etc/systemd/system/multi-user.target.wants/ceph-mon@{{ ansible_hostname }}.service"
```

Comment:
can you please use the 'file' module to create your symlink?
http://docs.ansible.com/ansible/file_module.html